testplan/report/testing/base.py

```diff
@@ -136,19 +136,19 @@ class ReportCategories(object):
     """
 
     TESTPLAN = "testplan"
-    TESTGROUP = "testgroup"
+    TESTGROUP = "testgroup"     # test group of unspecific type?
     MULTITEST = "multitest"
+    SUITE = "suite"
+    CASE = "case"
```

Comment:
This will break the UI since it expects testcases to have a category of "testcase". Please can you change back the value to "testcase"?

Reply:
Yea, i was debating whether I should change suite to testsuite or change testcase to case. I recon the latter one will have smaller impact as the CASE type is lately added by you recently. I prefer to keep the terminologies consistent, either testsuite/testcase or suite/case, but definitely not testsuite/case. We can do a global replace in the UI code from testcase -> case.

Reply:
as discussed, let's go with testsuite/testcase and be consistent everywhere please.